killbill-integration-tests/core/test_refund_credit.rb

```diff
@@ -308,5 +308,41 @@ def test_refund_and_cancel_with_item_adjustments
       assert_equal(refreshed_account.account_cba, 166.67)
     end
 
+    def test_find_credit_by_id
+
+      @child_account = create_child_account(@account)
```

Comment:
Don't create a child account here (since it's not a HA test), instead simply create a regular account.